src/vs/editor/browser/viewParts/indentGuides/indentGuides.css

```diff
@@ -3,13 +3,6 @@
  *  Licensed under the MIT License. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
 
-/*
-	Keeping name short for faster parsing.
-	cigr = core ident guides rendering (div)
-*/
-.monaco-editor .lines-content .cigr {
-	position: absolute;
-}
-.monaco-editor .lines-content .cigra {
```

Comment:
I don't really know how to measure it, but I really doubt that these abbreviations improve performance/memory consumption much. Every frame maybe renders at most up to 100 of those.